Vagrantfile

```diff
@@ -54,4 +54,45 @@ EOF
             end
         end
     end
+
+    # Ubuntu LTS nodes
+    (1..$num_instances).each do |n|
+        config.vm.define "ubuntu#{n}" do |node|
+            node.vm.box = "ubuntu/bionic64"
+            node.vm.network "private_network", ip: "192.168.64.#{n+20}"
+            node.vm.hostname = "ubuntu#{n}"
+
+            node.vm.provider "virtualbox" do |vb|
+                vb.name = "ubuntu#{n}"
+                vb.memory = "2048"
+                vb.cpus = "2"
+                vb.customize ["modifyvm", :id, "--audio", "none"]
+                vb.customize ["modifyvm", :id, "--memory", "2048"]
+                vb.customize ["modifyvm", :id, "--cpus", "2"]
+            end
+
+            node.vm.provision "shell" do |s|
+                s.inline = <<-SHELL
+
+                apt-get update
+                apt-get install -y ntp wget curl vim net-tools socat
+                echo 'sync time'
+                systemctl start ntp
+                systemctl enable ntp
+
+                echo 'set host name resolution'
+                cat >> /etc/hosts <<EOF
+192.168.64.21 ubuntu1
+192.168.64.22 ubuntu2
+192.168.64.23 ubuntu3
+192.168.64.24 ubuntu4
```

Comment:
the centos nodes should be here too, shouldn't they?

Reply:
Not intent to use mixed node setup.